### PR TITLE
Adding keymaps for iris and kyria keyboards

### PR DESCRIPTION
**Pull request by jhelvy**
## Description

I'm updating changes to my iris keymap, and I'm adding a kyria keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/jhelvy/keymap.c [hunk 1]
```diff
@@ -1,54 +1,137 @@
...
+        if (clockwise) {
+            tap_code(KC_MS_WH_RIGHT);
+        } else {
+            tap_code(KC_MS_WH_LEFT);
+        }
+    } else if (IS_LAYER_ON(MOUSE)) {
+        if (clockwise) {
+            register_code(KC_LGUI);
+            tap_code(KC_EQL);
+            unregister_code(KC_LGUI);
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/jhelvy/keymap.c:124]:
```suggestion
            tap_code16(G(KC_EQL));
```

File: keyboards/keebio/iris/keymaps/jhelvy/keymap.c [hunk 2]
```diff
@@ -1,54 +1,137 @@
...
+        } else {
+            register_code(KC_LGUI);
+            tap_code(KC_MINS);
+            unregister_code(KC_LGUI);
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/jhelvy/keymap.c:128]:
```suggestion
            tap_code16(G(KC_MINS));
```

File: keyboards/kyria/keymaps/jhelvy/keymap.c [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        if (clockwise) {
+            tap_code(KC_MS_WH_LEFT);
+        } else {
+            tap_code(KC_MS_WH_RIGHT);
+        }
+    } else if (IS_LAYER_ON(NUMBERS)) {
+        if (clockwise) {
+            register_code(KC_LGUI);
+            tap_code(KC_MINS);
+            unregister_code(KC_LGUI);
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/jhelvy/keymap.c:190]:
```suggestion
            tap_code16(G(KC_MINS));
```

File: keyboards/kyria/keymaps/jhelvy/keymap.c [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+        } else {
+            register_code(KC_LGUI);
+            tap_code(KC_EQL);
+            unregister_code(KC_LGUI);
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/jhelvy/keymap.c:194]:
```suggestion
            tap_code16(G(KC_EQL));
```

Comment (jhelvy) [reply keyboards/kyria/keymaps/jhelvy/keymap.c:194]: what do these additions do?

Comment (fauxpark) [reply keyboards/kyria/keymaps/jhelvy/keymap.c:194]: The same thing as the three lines you have, but in one line.

Comment (jhelvy) [reply keyboards/kyria/keymaps/jhelvy/keymap.c:194]: Oh cool, I was trying to figure out what the "G()" part was, but I guess that's the LGUI hold.

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**